### PR TITLE
maint: update .mailmap and AUTHORS

### PR DESCRIPTION
**Pull request by oscarbenjamin**
#### Brief description of what is fixed or changed

Update .mailmap and AUTHORS files.

#### Release Notes

NO ENTRY

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -1099,4 +1099,32 @@ Joannah Nanjekye <joannah.nanjekye@ibm.com>
 Ayush Kumar <ayushk7102@gmail.com>
 Kshitij <kshitijparwani.mat18@itbhu.ac.in>
 Daniel Hyams <dhyams@gmail.com>
+alijosephine <alijosephine@gmail.com>
 Matthias Köppe <mkoeppe@math.ucdavis.edu>
+mohajain <mohajain99@gmail.com>
+Anibal M. Medina-Mardones <ammedmar@gmail.com>
+Travis Ens <ens.travis@gmail.com>
+Evgenia Karunus <lakesare@gmail.com>
+RDxR10 <risirajdey@gmail.com>
```
Comment (oscarbenjamin) [inline AUTHORS:1108]: @RDxR10 would you like this to show your real name?

File: AUTHORS [hunk 2]
```diff
@@ -1099,4 +1099,32 @@ Joannah Nanjekye <joannah.nanjekye@ibm.com>
...
+Aman Thakur <54764701+jhonsnow456@users.noreply.github.com>
+S. Hanko <suzy.hanko@gmail.com>
+Dennis Sweeney <sweeney.427@osu.edu>
+gurpartap0306 <dhaliwal.gurpartap@gmail.com>
+Gurpartap Singh <36311440+gurpartap0306@users.noreply.github.com>
+Hampus Malmberg <hampus.malmberg88@gmail.com>
+scimax <max.kellermeier@hotmail.de>
+Nikhil Date <nikhil.s.date@gmail.com>
+Kuldeep Borkar Jr <kuldeepborkarjr765@gmail.com>
+AkuBrain <76952313+Franck2111@users.noreply.github.com>
```
Comment (oscarbenjamin) [inline AUTHORS:1128]: @Franck2111 would you like this to show your real name?

File: AUTHORS [hunk 3]
```diff
@@ -1099,4 +1099,32 @@ Joannah Nanjekye <joannah.nanjekye@ibm.com>
...
+Paul Spiering <paul@spiering.org>
+Pieter Gijsbers <p.gijsbers@tue.nl>
+Wang Ran (汪然) <wangr@smail.nju.edu.cn>
+naelsondouglas <naelson17@gmail.com>
+jhonsnow456 <thakuraman22july@gmail.com>
+Aman Thakur <54764701+jhonsnow456@users.noreply.github.com>
+S. Hanko <suzy.hanko@gmail.com>
+Dennis Sweeney <sweeney.427@osu.edu>
+gurpartap0306 <dhaliwal.gurpartap@gmail.com>
+Gurpartap Singh <36311440+gurpartap0306@users.noreply.github.com>
```
Comment (oscarbenjamin) [inline AUTHORS:1123]: @gurpartap0306 would you like this to list a different email address?

File: AUTHORS [hunk 4]
```diff
@@ -1099,4 +1099,30 @@ Joannah Nanjekye <joannah.nanjekye@ibm.com>
...
+alijosephine <alijosephine@gmail.com>
 Matthias Köppe <mkoeppe@math.ucdavis.edu>
+mohajain <mohajain99@gmail.com>
+Anibal M. Medina-Mardones <ammedmar@gmail.com>
+Travis Ens <ens.travis@gmail.com>
+Evgenia Karunus <lakesare@gmail.com>
+RDxR10 <risirajdey@gmail.com>
+lastcodestanding <rohang71604@gmail.com>
+Andrey Lekar <andrey_lekar@adoriasoft.com>
+Abbas <42001049+iam-abbas@users.noreply.github.com>
```
Comment (oscarbenjamin) [inline AUTHORS:1111]: @iam-abbas do you want a different name and email address to be recorded here?

Comment (iam-abbas) [reply AUTHORS:1111]: Can we have email to iamabbas7@gmail.com

Comment (oscarbenjamin) [reply AUTHORS:1111]: Yes. Are you happy for your name to be recorded as just "Abbas" or would you prefer something else?

File: AUTHORS (hunk 1 above)
Comment (RDxR10) [reply AUTHORS:1108]: I would prefer "Risiraj Dey". Thanks

File: AUTHORS (hunk 4 above)
Comment (iam-abbas) [reply AUTHORS]: Let's do `Abbas Mohammed`. Sorry for being in decisive ahaha. This is final.

Comment (oscarbenjamin) [reply AUTHORS]: Thanks, I've updated that.

File: AUTHORS (hunk 1 above)
Comment (oscarbenjamin) [reply AUTHORS]: Thanks. I've change that.

**Comment by oscarbenjamin** [comment]
There is a failure here in the optional dependency tests:
```
_____________ sympy/polys/tests/test_ring_series.py:test_nth_root ______________
Traceback (most recent call last):
  File "/home/runner/work/sympy/sympy/sympy/polys/tests/test_ring_series.py", line 232, in test_nth_root
    assert rs_nth_root(8*x, 3, x, 3) == 2*x**QQ(1, 3)
  File "/home/runner/work/sympy/sympy/sympy/polys/ring_series.py", line 988, in rs_nth_root
    const = R(c**Rational(1, n))  # exponentiation with mpq object
TypeError: unsupported operand type(s) for ** or pow()
```
That's definitely not caused by the changes here. Not sure if it's a random failure or a problem on master.

File: AUTHORS [hunk 5]
```diff
@@ -1099,4 +1099,30 @@ Joannah Nanjekye <joannah.nanjekye@ibm.com>
...
+Evgenia Karunus <lakesare@gmail.com>
+Risiraj Dey <risirajdey@gmail.com>
+lastcodestanding <rohang71604@gmail.com>
+Andrey Lekar <andrey_lekar@adoriasoft.com>
+Abbas Mohammed <42001049+iam-abbas@users.noreply.github.com>
+anutosh491 <andersonbhat491@gmail.com>
+Steve Kieffer <sk@skieffer.info>
+Paul Spiering <paul@spiering.org>
+Pieter Gijsbers <p.gijsbers@tue.nl>
+Wang Ran (汪然) <wangr@smail.nju.edu.cn>
```
Comment (WrRan) [inline AUTHORS:1116]: The information is right.

Comment (WrRan) [inline AUTHORS:1116]: The information is right.

**Pull request merged**